charts/ai-model/templates/ai-model-ingress.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
```

Comment:
Cannot be hard coded like this
Annotations must come from values file (look at other chart for examples)

Reply:
Moved to values